server/tests/audit_math/test_bravo.py

```diff
@@ -598,3 +679,53 @@ def test_tied_contest():
         "all-ballots": {"type": "all-ballots", "size": 4992420, "prob": None},
     },
 }
+
+
+round2_sample_sizes = {
```

Comment:
it kind of seems like all these round 2 sample sizes that are negative shouldn't be allowed. maybe it's not easy to disallow that though?

at least an explanatory comment would be helpful

Reply:
I fixed it so that no negative sample sizes will get returned, and cases where it would have happened result in errors being thrown.